### PR TITLE
Add description field to all input type components.

### LINKED ISSUE #11: 'Description' property for each component
Currently, the only way to display a description for a component is to use the 'text' component but it doesn't look good.
I would like to use a 'description' that would be displayed under the component with a smaller font.

Do you see a better solution ? I tried to play with html in the "label" but the <em> <small> <i> ... tags don't seem to work

### PR DESCRIPTION
**Pull request by keegan-lillo**
Resolves #11
I'm not super happy with the tap-highlight now that the items can potentially be taller. This will be easier to fix once #23 gets merged

### CODE REVIEW

**Comment by keegan-lillo** [comment]
Okay fixed the tap-highlight so it only highlights the interactive part of the item.

File: src/styles/clay/_base.scss [hunk 1]
```diff
@@ -103,15 +104,13 @@ label {
...
+    padding: $item-spacing-v $item-spacing-h;
 
-    &:last-child,
-    &:first-child {
-      padding-bottom: $item-spacing-v;
-    }
+    //&:last-child,
+    //&:first-child {
+    //  padding-bottom: $item-spacing-v;
+    //}
```
Comment (matthewtole) [inline src/styles/clay/_base.scss]: Remove commented out lines

File: src/styles/clay/_base.scss [hunk 2]
```diff
@@ -154,52 +153,59 @@ a {
...
+  //      padding-right: 1em;
+  //
+  //      @include placeholder {
+  //        color: $color-orange;
+  //      }
+  //
+  //    }
+  //  }
+  //
+  //}
```
Comment (matthewtole) [inline src/styles/clay/_base.scss]: Remove commented out lines

**Comment by matthewtole** [comment]
Update README screenshot with one or two examples of these new descriptions.

After that, :+1:

**Pull request merged**